linera-views/src/views/map_view.rs

```diff
@@ -1279,7 +1367,7 @@ where
     {
         let prefix = Vec::new();
         self.map
-            .for_each_key_value(
+            .for_each_key_value_or_bytes(
```

Comment:
Most of these can be just `for_each_key_value`, so we don't need the `to_value` call below.